### PR TITLE
Use preparer in production

### PR DESCRIPTION
**Pull request by dynamic-entropy**
Use `conveyor-preparer` in production.

### CODE REVIEW

File: apps/base/rucio-daemons/cms-rucio-daemons.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ image:
 
 abacusAccountCount: 1
 abacusRseCount: 1
-#conveyorTransferSubmitterCount: 2
+conveyorPreparerCount: 3
```
Comment (ericvaandering) [inline apps/base/rucio-daemons/cms-rucio-daemons.yaml:6]: Why three of these?

File: apps/production/prod-rucio-daemons.yaml [hunk 2]
```diff
@@ -16,9 +16,6 @@ reaper:
   chunkSize: 4000
   sleepTime: 120
 
-conveyorTransferSubmitter:
```
Comment (ericvaandering) [inline apps/production/prod-rucio-daemons.yaml:19]: Why remove this?

**Review by ericvaandering — CHANGES_REQUESTED** [review]

File: apps/base/rucio-daemons/cms-rucio-daemons.yaml (hunk 1 above)
Comment (dynamic-entropy) [reply apps/base/rucio-daemons/cms-rucio-daemons.yaml:6]: I made it the same as the conveyor Submitter count we have

File: apps/production/prod-rucio-daemons.yaml (hunk 2 above)
Comment (dynamic-entropy) [reply apps/production/prod-rucio-daemons.yaml:19]: Moved it to base config file

File: apps/base/rucio-daemons/cms-rucio-daemons.yaml [hunk 3]
```diff
@@ -93,6 +93,13 @@ conveyorTransferSubmitter:
   additionalEnvs:
     - name: X509_USER_PROXY
       value: "/opt/proxy/x509up"
+  threads: 5
```
Comment (dynamic-entropy) [inline apps/base/rucio-daemons/cms-rucio-daemons.yaml:96]: Here is the thread config for the submitter, this keeps all its config in a single place

**Review by dynamic-entropy — COMMENTED** [review]
I have tried to consolidate the config for the submitter

**Comment by dciangot** [comment]
Since Eric is on vacation, but looks like an "urgent" request (@haozturk can confirm), I propose we go ahead with the current changes and @dynamic-entropy keeps an eye in case anything goes wrong.

**Comment by haozturk** [comment]
Thanks @dciangot indeed, we really need this to be able to deploy conveyor-throttler, which I'd like to deploy tomorrow if this one goes well. Currently, we're holding off a large transfer backlog to a few sites. Changes look good to me. I'll go ahead with merging it. We should address Eric's review when he's back.

**Pull request merged**